.circleci/config.yml

```diff
@@ -50,9 +50,31 @@ jobs:
       - run:
           name: Check for changes
           command: git diff --exit-code
+  check-forge-fmt:
+    docker:
+      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:latest
+    steps:
+      - checkout
+      - run:
+          name: check compliance with forge fmt
+          command: |
+            forge fmt --check
+  check-security-configs:
+    circleci_ip_ranges: true
+    docker:
+      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:latest
+    steps:
+      - checkout
+      - run:
+          name: check security configs
+          command: |
+            forge build
+            forge script CheckSecurityConfigs --rpc-url="https://ci-mainnet-l1.optimism.io"
```

Comment:
Since this is a foundry project, we should update the gitignore to match the foundry default, which we can do by adding the below. (Currently running `forge build` results in git detecting changes to the `cache/` and `out/` folders which we don't want)

```
# Compiler files
cache/
out/

# Ignores development broadcast logs
!/broadcast
/broadcast/*/31337/
/broadcast/**/dry-run/

# Docs
docs/
```